Sift/Schema/ComplexTypes/wire.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Wire",
+  "type": "object",
+  "required": [ "$wire_type", "$account_holder_name"],
```

Comment:
`routing_number` is also a required field. you can add it here.

Reply:
Customer should send one required -  either routing or swift or bic

Reply:
oh yes, i missed that.